packages/core/src/utilities/updateVTKImageDataWithCornerstoneImage.ts

```diff
@@ -0,0 +1,39 @@
+import vtkImageData from '@kitware/vtk.js/Common/DataModel/ImageData';
+import { IImage } from '../types';
+
+function updateVTKImageDataWithCornerstoneImage(
+  sourceImageData: vtkImageData,
+  image: IImage
+) {
+  const pixelData = image.getPixelData();
+  const scalarData = sourceImageData.getPointData().getScalars().getData() as
+    | Uint8Array
```

Comment:
Could you declare this somewhere - there are so many times it gets used that I'd rather have a specific name for it.